ai_server/server.py

```diff
@@ -12,15 +12,53 @@
 
 @app.route('/parse_description', methods=['POST'])
 def parse_description():
+
     description = request.json.get('description')
 
+    # class content should be a copy paste of the WorldInfo.cs file
```

Comment:
Wait this is stupid, just read the file into a string!!!

Reply:
i thought of this too, but when you build the project say for a for webgl or just for a desktop application, the .cs file will be compiled and you wont be able to read the raw file anymore

Reply:
What if you just made the prompt smarter by giving it specific requirements instead of just handing it the entire code.